plugins/services/src/js/components/forms/ContainerServiceFormAdvancedSection.js

```diff
@@ -105,7 +105,7 @@ class ContainerServiceFormAdvancedSection extends Component {
     if (gpusDisabled) {
       inputNode = (
         <Tooltip
-          content="Docker Engine does not support GPU resources, please select Universal Container Runtime if you want to use GPU resources."
+          content="Docker Engine does not support GPU resources, please select Mesos Runtime if you want to use GPU resources."
```

Comment:
Sorry for not addressing this! I opted not to for this PR to keep it as simple as possible, but I agree this should be internationalized.